resources/views/accessories/view.blade.php

```diff
@@ -88,17 +82,154 @@ class="table table-striped snipe-table"
-      </div>
+    @if ($accessory->image!='')
+        <div class="col-md-12 text-center" style="padding-bottom: 15px;">
+            <a href="{{ app('accessories_upload_url') }}{{ $accessory->image }}" data-toggle="lightbox"><img src="{{ app('accessories_upload_url') }}{{ $accessory->image }}" class="img-responsive img-thumbnail" alt="{{ $accessory->name }}"></a>
+        </div>
+    @endif
+
+
+
+    <div class="col-md-12">
+        <ul class="list-unstyled" style="line-height: 25px; padding-bottom: 20px;">
+            @if ($accessory->model_number!='')
+            <li>
+                <strong>{{ trans('general.model_no') }}: </strong>
+                {{ $accessory->model_number }}
+            </li>
+            @endif
+
+            @if ($accessory->company)
+            <li>
+                <strong>{{ trans('general.company') }}: </strong>
+                <a href="{{ route('companies.show', $accessory->company->id) }}">{{ $accessory->company->name }}</a>
+            </li>
+            @endif
+            
+            @if ($accessory->department)
+            <li>
+                <strong>{{ trans('general.department') }}: </strong>
+                <a href="{{ route('departments.show', $accessory->department->id) }}">{{ $accessory->department->name }}</a>
+            </li>
+            @endif
+
+            @if ($accessory->category)
+            <li>
+                <strong>{{ trans('general.category') }}: </strong>
+                @can('view', \App\Models\Category::class)
+                  <a href="{{ route('categories.show', $accessory->category->id) }}">{{ $accessory->category->name }}</a>
+                @else
+                  {{ $accessory->category->name }}
+                @endcan
+            </li>
+            @endif
+
+            @if ($accessory->location)
+            <li><strong>{{ trans('general.location') }}: </strong>
+                @can('superuser')
+                    <a href="{{ route('locations.show', ['location' => $accessory->location->id]) }}">
+                    {{ $accessory->location->name }}
+                    </a>
+                @else
+                    {{ $accessory->location->name }}
+                @endcan
+            </li>
+            @endif
+
+            <br />
+            <li>
+                <strong>{{ trans('admin/accessories/general.total') }}: </strong>
+                {{ $accessory->qty }}
+            </li>
+            <li>
+                <strong>{{ trans('general.checkouts_count') }}: </strong>
+                {{ $accessory->qty - $accessory->numRemaining() }}
+            </li>
+            <li>
+                <strong>{{ trans('admin/accessories/general.remaining') }}: </strong>
+                {{ $accessory->numRemaining() }}
+            </li>
+            <li>
+                <strong>{{ trans('general.min_amt') }}: </strong>
+                {{ $accessory->min_amt }}
+            </li>
+
+
+            @if ($accessory->supplier)
+            <br />
```

Comment:
A lot of supplier details here, are we sure we want that? I'm not sure. I'm not totally against that, mind you. Just pondering out loud.